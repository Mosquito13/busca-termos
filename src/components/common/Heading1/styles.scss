.heading-1 {
  font-family: 'Oregano', cursive;
  font-size: 6rem;
  font-weight: 400;
  color: currentColor;
  line-height: 1;
}
