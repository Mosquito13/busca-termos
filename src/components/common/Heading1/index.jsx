import './styles.scss';

const Heading1 = ({ children }) => {
  return <h1 className='heading-1'>{children}</h1>;
};

export default Heading1;
