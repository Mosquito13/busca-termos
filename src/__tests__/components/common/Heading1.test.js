import renderer from 'react-test-renderer';

import Heading1 from '../../../components/common/Heading1';

describe(`${Heading1.name}`, () => {
  describe('snapshots', () => {
    it('should render', () => {
      const tree = renderer
        .create(<Heading1 value="value for heading" />)
        .toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
