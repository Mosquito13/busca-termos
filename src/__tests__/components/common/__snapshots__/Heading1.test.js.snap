// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading1 snapshots should render 1`] = `
<h1
  className="heading-1"
>
  value for heading
</h1>
`;
